packages/components/atoms/f-button/test/component/f-button.component.spec.js

```diff
@@ -13,7 +13,18 @@ describe('f-button component tests', () => {
         button.load();
 
         // Assert
-        expect(button.isActionComponentDisplayed()).toBe(true);
+        expect(button.isComponentDisplayed()).toBe(true);
+    });
+
+    it('should check that the f-button action component is clickable', () => {
+        // Arrange
+        button = new Button();
```

Comment:
When you update  the file name, change this to `button = new ActionButton();` to match the naming convention.